plugins/in_syslog/syslog_prot.c

```diff
@@ -104,7 +104,10 @@ int syslog_prot_process(struct syslog_conn *conn)
             flb_free(out_buf);
         }
         else {
-            flb_warn("[in_syslog] error parsing log message");
+            flb_warn("[in_syslog] error parsing log message "
+                    "(input_plugin_alias_name='%s' parser_name='%s')",
+                     ctx->i_ins->alias, ctx->parser->name);
+            flb_debug("[in_syslog] unparsed log message: %s", out_buf);
```

Comment:
the debug message might lead to a crash. The variable ```out_buf``` contains the parsed message in binary format if flb_parser_do() succeed. The log message that failed is pointed by ```p``` with length ```len```.